configure/migrator/migrator.Job.yaml

```diff
@@ -0,0 +1,184 @@
+#             value: "sg"
+#           - name: CODEINTEL_PGPASSWORD
+#             value: "sg"
+#           - name: CODEINTEL_PGDATABASE
+#             value: "sg"
+#           - name: CODEINTEL_PGSSLMODE
+#             value: "disable"
+#           - name: CODEINSIGHTS_PGHOST
+#             value: "codeinsights-db"
+#           - name: CODEINSIGHTS_PGPORT
+#             value: "5432"
+#           - name: CODEINSIGHTS_PGUSER
+#             value: "postgres"
+#           - name: CODEINSIGHTS_PGPASSWORD
+#             value: "password"
+#           - name: CODEINSIGHTS_PGDATABASE
+#             value: "postgres"
+#           - name: CODEINSIGHTS_PGSSLMODE
+#             value: "disable"
+#       restartPolicy: OnFailure
+#   backoffLimit: 4
+# ---
+# apiVersion: batch/v1
+# kind: Job
+# metadata:
+#   name: migrator-frontend
+#   annotations:
+#     description: Database migrator for the frontend database
+#   labels:
+#     app.component.kubernetes/component: migrator
+#     deploy: sourcegraph
+# spec:
+#   template:
+#     spec:
+#       containers:
+#       - name: migrator
+#         image: "sourcegraph/migrator:3.36.0"
+#         args: ["up", "-db", "frontend"]
+#         env:
+#           - name: PGHOST
+#             value: "pgsql"
+#           - name: PGPORT
+#             value: "5432"
+#           - name: PGUSER
+#             value: "sg"
+#           - name: PGPASSWORD
+#             value: "sg"
+#           - name: PGDATABASE
+#             value: "sg"
+#           - name: PGSSLMODE
+#             value: "disable"
+#       restartPolicy: OnFailure
+#   backoffLimit: 4
+# 
+# ---
+# apiVersion: batch/v1
+# kind: Job
+# metadata:
+#   name: migrator-codeintel
+#   annotations:
+#     description: Database migrator for the codeintel database
+#   labels:
+#     app.component.kubernetes/component: migrator
+#     deploy: sourcegraph
+# spec:
+#   template:
+#     spec:
+#       containers:
+#       - name: migrator
+#         image: "sourcegraph/migrator:3.36.0"
+#         args: ["up", "-db", "codeintel"]
+#         env:
+#           - name: CODEINTEL_PGHOST
+#             value: "codeintel-db"
+#           - name: CODEINTEL_PGPORT
+#             value: "5432"
+#           - name: CODEINTEL_PGUSER
+#             value: "sg"
+#           - name: CODEINTEL_PGPASSWORD
+#             value: "sg"
+#           - name: CODEINTEL_PGDATABASE
+#             value: "sg"
+#           - name: CODEINTEL_PGSSLMODE
+#             value: "disable"
+#       restartPolicy: OnFailure
+#   backoffLimit: 4
+#
+# ---
+# apiVersion: batch/v1
+# kind: Job
+# metadata:
+#   name: migrator-codeinsights
+#   annotations:
+#     description: Database migrator for codeinsights db
+#   labels:
+#     app.component.kubernetes/component: migrator
+#     deploy: sourcegraph
+# spec:
+#   template:
+#     spec:
+#       containers:
+#       - name: migrator
+#         image: "sourcegraph/migrator:3.35.1"
+#         args: ["up", "-db", "codeinsights"]
+#         env:
```

Comment:
If we do end up keeping these examples, this `env` list can be trimmed down to just CodeInsights related stuff.

Reply:
There's currently an issue requiring having all the environments configured to be able to migrate `codeinsights-db`: https://github.com/sourcegraph/sourcegraph/issues/30013

Reply:
Nice find!